src/App/Action/Backend/Order/ExportOrderAction.php

```diff
@@ -97,6 +106,13 @@ protected function exportOrders(PdkOrderCollection $orders): PdkOrderCollection
 
         $apiOrders = $this->orderRepository->postOrders($fulfilmentOrders);
 
+        $apiOrders->each(function (Order $order) {
```

Comment:
Dit moet in een losse action gebeuren, zo kan je deze handeling nu ook niet los aanroepen vanuit bijvoorbeeld een hook in WooCommerce